dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantMemoryStore.cs

```diff
@@ -0,0 +1,369 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.AI.Embeddings;
+using Microsoft.SemanticKernel.Connectors.Memory.Qdrant.Diagnostics;
+using Microsoft.SemanticKernel.Memory;
+
+namespace Microsoft.SemanticKernel.Connectors.Memory.Qdrant;
+
+/// <summary>
+/// An implementation of <see cref="IMemoryStore"/> for Qdrant Vector database.
+/// </summary>
+/// <remarks>The Embedding data is saved to a Qdrant Vector database instance specified in the constructor by url and port.
+/// The embedding data persists between subsequent instances and has similarity search capability.
+/// </remarks>
+public class QdrantMemoryStore : IMemoryStore
+{
+    /// <summary>
+    /// Constructor for a memory store backed by a Qdrant Vector database instance.
+    /// </summary>
+    /// <param name="host"></param>
```

Comment:
Same comment re:adding parameter descriptions or removing tags.

Reply:
Or just do <inheritdoc/> here after filling those up in the interface...